src/components/Landing/Landing.tsx

```diff
@@ -20,17 +21,21 @@ export const Landing: React.FC = () => {
     useSectionScroll();
 
     return (
-        <PageConstructor
-            content={(pathname === '/rtl' ? getRtlLanding(t) : getLanding(t)) as PageContent}
-            custom={{
-                blocks: {
-                    [CustomBlock.CustomHeader]: CustomHeader,
-                    [CustomBlock.CustomExtendedFeatures]: CustomExtendedFeatures,
-                    [CustomBlock.Examples]: Examples,
-                    [CustomBlock.Roadmap]: RoadmapBlock,
-                    [CustomBlock.Templates]: TemplatesBlock,
-                },
-            }}
-        />
+        <>
+            <GithubStarsBlock device="desktop" />
```

Comment:
Let's hide this block for rtl landing or adapt it for rtl.

Reply:
fixed both comments